CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- Fix: typeInformation contain global config values (#1515)
```

Comment:
Taking into account this is an internal refactor to simplify code more than an actual fix, I'd suggest to use this wording better:

```suggestion
- Hardening: simplify implementation so typeInformation contain global config values (#1515)
```